substrate/bin/utils/chain-spec-builder/src/lib.rs

```diff
@@ -28,60 +28,94 @@
 //! See [`ChainSpecBuilderCmd`] for a list of available commands.
 //!
 //! ## Typical use-cases.
-//! ##### Get default config from runtime.
+//! ##### Generate chains-spec using default config from runtime.
 //!
-//! Query the default genesis config from the provided `runtime.wasm` and use it in the chain
-//! spec. The tool allows specifying where to write the chain spec, and optionally also where the
-//! write the default genesis state config (which is `/dev/stdout` in the following example):
-//! ```text
-//! chain-spec-builder --chain_spec_path ./my_chain_spec.json create -r runtime.wasm default /dev/stdout
+//!	Query the default genesis config from the provided `runtime.wasm` and use it in the chain
+//! spec.
+//!	```bash
+//! chain-spec-builder create -r runtime.wasm default
 //! ```
-//!
-//! _Note:_ [`GenesisBuilder::create_default_config`][sp-genesis-builder-create] runtime function is
+//! 
+//! _Note:_ [`GenesisBuilder::get_preset`][sp-genesis-builder-get-preset] runtime function is
 //! called.
 //!
 //!
```

Comment:
I've always wondered if we can use `docify` to document CLIs or not. The only reasonable hack is to write a cli-test that performs an action eg `chain-spec-builder display-preset -r runtime.wasm` then display it here.